Update README.md

panuoksala opened the pull request:
Add Stream Deck plugin and oksalanet url

johnlokerse commented:
Thanks for the contribution @panuoksala. Can you add a section "Azure DevOps Tools and Extensions" under "Community content" and add your streamdeck extension there?

panuoksala commented:
I added "Community and Extensions" topic, because I think that Azure DevOps is kind of pointless in that context (The whole page is about that)

Added back to top link + top link also.
